config/local-setup/controller/kustomization.yaml

```diff
@@ -5,24 +5,25 @@ namePrefix: mgc-
 resources:
   - namespace.yaml
   - managed_zone.yaml
+  - dnsprovider.yaml
 
 generatorOptions:
   disableNameSuffixHash: true
 
 configMapGenerator:
-  - name: controller-config
+  - name: provider-config
     envs:
-      - ../../../controller-config.env
+      - ../../../dns-provider-config.env
 
 secretGenerator:
-  - name: aws-credentials
+  - name: provider-credentials
     envs:
-      - ../../../aws-credentials.env
+      - ../../../provider-credentials.env
```

Comment:
There's really no need to change these expected file names. Whatever way you eventually do this, you will be creating something with "AWS" credentials in it for an AWS provider, so "aws-credentials.env" make more sense anyway.